ipywidgets_jsonschema/form.py

```diff
@@ -931,12 +931,19 @@ def _resetter():
         # Initially call the resetter
         _resetter()
 
+        def _getter():
+            result = [h.getter() for h in elements[:element_size]]
+            if schema.get("set", False):
+                if len(result) != len(set(result)):
+                    raise FormError("Array elements are not unique")
```

Comment:
Looking at what `pydantic` does, I think it would also be acceptable to make the array unique instead of throwing an error:

```python
import pydantic
from typing import Set

class Foo(pydantic.BaseModel):
    ids: Set[str]

Foo(ids=["a", "a"])
```

above code outputs `Foo(ids={'a'})`